init.ps1

```diff
@@ -81,6 +95,78 @@ function CreateDirectory($message, $path){
     New-Item -Force -ItemType Directory -Path $path | Out-Null
 }
 
+function Copy-GithubRepository

```

Comment:
That's a good longer term approach.  I wanted to get something viable without having to publish and host nuget packages for now.